src/api/SearchApi.js

```diff
@@ -65,6 +66,48 @@ const SEARCH_TERM :string = 'searchTerm';
 const START :string = 'start';
 
 /**
+ * `GET /search/home/{start}/{maxHits}`
+ *
+ * Executes a search over all existing entity sets to populate the home page
+ *
+ * @static
+ * @memberof loom-data.SearchApi
+ * @param {number} start
+ * @param {number} maxHits
+ * @returns {Promise}
+ *
+ * @example
+ * SearchApi.loadHomePageEntitySets(0, 10);
+ */
+
+export function loadHomePageEntitySets(start :number, maxHits :number) :Promise<> {
```

Comment:
Well, what does the new endpoint do / return? Also, is it a "load all" endpoint, or a "search" endpoint? If it's a "load all" endpoint, doesn't it belong in `EntityDataModelApi.js`?